app/integration/project-generator/src/main/java/io/syndesis/integration/project/generator/ProjectGenerator.java

```diff
@@ -51,30 +71,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
-import java.io.PipedInputStream;
-import java.io.PipedOutputStream;
-import java.net.URISyntaxException;
-import java.net.URL;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Properties;
-import java.util.Set;
-import java.util.TreeSet;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.stream.Collectors;
-
-import static io.syndesis.integration.project.generator.ProjectGeneratorHelper.addResource;
-import static io.syndesis.integration.project.generator.ProjectGeneratorHelper.addTarEntry;
-import static io.syndesis.integration.project.generator.ProjectGeneratorHelper.compile;
-import static io.syndesis.integration.project.generator.ProjectGeneratorHelper.mandatoryDecrypt;
+import static io.syndesis.integration.project.generator.ProjectGeneratorHelper.*;
```

Comment:
guess this file has been changed by mistake, afaik, we do not use star import.

Reply:
It was a mistake, fixed, thanks.